mirna_explorer/repository/create_database.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import os
+import sqlite3
+
+data_folder = "../../../mirna_target_prediction/data/"
+
+required_files = ["raw/UTR_Sequences.txt",
+				  "processed/mirna_id_dictionary.tsv",
+				  "processed/gene_id_dictionary.tsv",
+				  "processed/scored_interactions_processed.tsv"]
+
+for filename in required_files:
+	path = data_folder + filename
+	if not os.path.isfile(path):
+		print(f'error: file not found: \'{path}\', aborting')
+		exit(1)
+
+db_path = 'database/db.sqlite'
```

Comment:
Constants should be at the top of the file and capitalized, ie. `DB_PATH`

Reply:
Let's give our boy `db.sqlite` some love and give him a better birth name like, uhm, mirna_db.sqlite :dancer: